ui/app/AppLayouts/Profile/Sections/Ens/List.qml

```diff
@@ -287,6 +287,13 @@ Item {
                 anchors.leftMargin: chatBox.chatHorizontalPadding
                 width: parent.width
                 anchors.right: parent.right
+                Connections {
```

Comment:
In this section, the text will never change right, since it's a placeholder? Maybe we don't need the connection